spec/models/manageiq/providers/vmware/cloud_manager_spec.rb

```diff
@@ -101,4 +101,20 @@
   it "#supported_catalog_types" do
     expect(@ems.supported_catalog_types).to eq(%w(vmware))
   end
+
+  describe 'snapshot operations' do
+    let(:vm) { FactoryGirl.create(:vm_vmware, :ext_management_system => @ems) }
+
+    context ".vm_create_snapshot" do
+      let(:snapshot_options) { {:name => 'name', :memory => false, :quiesce => false} }
```

Comment:
Why are you passing `:quiesce` here? I think it's never used then, because it's hard-coded to "false"